package.json

```diff
@@ -36,19 +36,19 @@
     "dep:update": "npx ncu -u"
   },
   "dependencies": {
-    "knex": "0.21.2",
-    "lodash": "4.17.19",
+    "knex": "0.21.12",
+    "lodash": "4.17.20",
     "mysql": "2.18.1"
   },
   "devDependencies": {
     "faucet": "0.0.1",
     "nyc": "15.1.0",
-    "npm-audit-resolver": "2.2.0",
-    "npm-check-updates": "7.0.1",
+    "npm-audit-resolver": "2.2.1",
+    "npm-check-updates": "10.2.1",
     "pre-commit": "1.2.2",
     "proxyquire": "2.1.3",
-    "sinon": "9.0.2",
-    "standard": "14.3.4",
+    "sinon": "9.2.1",
+    "standard": "16.0.3",
```

Comment:
is the standard passing with that change?
has this been resolved?
https://github.com/mojaloop/project/issues/1849

Reply:
@vgenev  Yes, it is.